setup/quickstart/index.md

```diff
@@ -8,7 +8,7 @@ sidebar:
 Here are a few quickstart solutions. These are not meant for production use. To install spinnaker for more than just evaluation, you need a [full install](/setup/install/) using Halyard.
 
 * [Amazon Web Services](https://aws.amazon.com/about-aws/whats-new/2016/08/netflix-oss-spinnaker-on-the-aws-cloud-quick-start-reference-deployment/)
-* [Google Cloud Launcher](https://cloud.google.com/launcher/solution/click-to-deploy-images/spinnaker?q=spinnaker) 
+* [Spinnaker for Google Cloud Platform](https://github.com/googlecloudplatform/spinnaker-for-gcp)
```

Comment:
Perhaps point to the documentation page rather than the repo?

Reply:
:+1: